packages/falso/src/tests/song.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { randSong } from '../lib/song';
+
+describe('song', () => {
+  it('should start with a capital letter', () => {
+    const result = randSong();
+    const firstChar = result.charAt(0);
+
+    expect(firstChar).toEqual(firstChar.toUpperCase());
+  });
+
+  it('has no leading and trailing spaces', () => {
+    const song = randSong();
+
+    expect(song[0]).not.toBe(' ');
+    expect(song[song.length - 1]).not.toBe(' ');
+  });
+
+  // it('should create', () => {
```

Comment:
Why in comment?

Reply:
Hey, that can be removed. It was in the template and I just kept it there.